README-development.md

```diff
@@ -11,35 +11,16 @@
 - mock
 
 ### May need to be built from source
-- [coreos-assembler](https://github.com/cgwalters/coreos-assembler/): [container](http://quay.io/cgwalters/coreos-assembler)
 - [rpmdistro-gitoverlay](https://github.com/projectatomic/rpmdistro-gitoverlay)
 - [imagefactory-plugins-TinMan](https://github.com/redhat-imaging/imagefactory)
 
 ## Building
 
 - Clone ``openshift/os``
 - Move into the cloned repo
-- Build packages from source repos:
- - ``rpmdistro-gitoverlay init``
- - ``rpmdistro-gitoverlay resolve --fetch-all``
- - ``rpmdistro-gitoverlay build``
-- Make the ostree
- - ``export REPO=/tmp/repo``
- - ``ostree init --repo=$REPO --mode=archive``
- - ``podman run --privileged --rm -v $REPO:/srv quay.io/cgwalters/coreos-assembler``
- - ``cd /srv && coreos-assembler --repo=/srv --cachedir=cache host.yml``
- - ``exit``
- - ``ostree summary --repo=$REPO --update``
-- Make the qcow2
- - ``ostree --repo=repo remote add rhcos --no-gpg-verify ${OSTREE_INSTALL_URL}``
- - ``ostree --repo=repo pull --mirror --commit-metadata-only rhcos``
- - ``sed -i 's,\(<url>\).*\(<\/url\),\1${INSTALLER_TREE_URL}\2,' rhcos.tdl``
- - ``sed -i 's,@@OSTREE_INSTALL_URL@@,${OSTREE_INSTALL_URL},' cloud.ks``
- - ``export REF="openshift/3.10/x86_64/os"``
- - ``sed -i 's,@@OSTREE_INSTALL_REF@@,${REF},' cloud.ks``
- - ``imagefactory --debug base_image --file-parameter install_script cloud.ks --parameter offline_icicle True rhcos.tdl``
- - ``export IMAGE=`ls /var/lib/imagefactory/storage/*.body```
- - ``qemu-img convert -f raw -O qcow2 $IMAGE rhcos-devel.qcow2``
+- Build packages from source repos: ``make rdgo``
+- Make the ostree: ``OSTREE_INSTALL_URL=<OSTREE_INSTALL_URL_HERE> make rpmostree-compose``
```

Comment:
Minor: I think the more standard way of setting vars when calling make is `make <target> VAR1=val1 VAR2=val2...`.